docs/release-notes/8.15.asciidoc

```diff
@@ -224,6 +224,20 @@ On September 5, 2024, this issue was resolved.
 ====
 // end::known-issue-14686[]
 
+// tag::known-issue-crowdstrike-response-actions[]
+[discrete]
+.CrowdStrike response actions (isolate and release host) not working
+[%collapsible]
+====
+*Details* +
+A bug prevented third-party response actions with CrowdStrike from working.
+
+*Workaround* +
+Upgrade to 8.15.1 or later.
```

Comment:
I just noticed that this was added to the 8.15.1 release notes. Should it be doc'd in the 8.15.0 instead?

Reply:
Hm, I think this _is_ the 8.15.0 section, because that's where the bug/known issue exists. I think I might need to add an item for the bug _fix_ to the 8.15.1 section above.

Reply:
Nvm, you're right! I didn't expand this file enough when checking your entry's location. The known issue and bug fix entries look good 👍🏼